backend/models/tag_model.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel, Field
+from enum import Enum
+from typing import List
+from datetime import datetime
+from typing import Optional
+
+
+class Tag(BaseModel):
+    id: int
+    content: str = Field(
+        ..., max_length=600, description="content associated with the tag"
+    )
+    created_at: datetime | None = None
```

Comment:
For consistency: either `created_at: Optional[datetime]` or `created_at: datetime | None = None`.